rootfs_linux.go

```diff
@@ -62,6 +74,13 @@ func setupRootfs(config *configs.Config, console *linuxConsole) (err error) {
 	return nil
 }
 
+func mountCmd(cmd []string) error {
```

Comment:
I think we should check for length of `cmd` here, it should not panic if cmd is empty